### DISCUSSION TITLE
Swagger API v2

### DISCUSSION THREAD

**Discussion by ThongVM003** [edgexfoundry/community#301] [Q&A] [answered]
Hello, everybody. I need help with how can I open the swagger API v2 edgex. I can not open  "https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.0.0#". Thank all !

**Comment by cloudxxx8** [comment, marked as answer]
we moved all the APIs to our docs website: https://docs.edgexfoundry.org/3.1/api/core/Ch-APICoreMetadata/
by the way, v2 is out of support, but you can still retrieve the files from the source code https://github.com/edgexfoundry/edgex-go/tree/v2.3.0/openapi/v2
You use any Swagger editor or viewer to read files
